front/app/containers/IdeasNewPage/WithJSONForm/index.tsx

```diff
@@ -113,10 +115,23 @@ const IdeasNewPageWithJSONForm = ({ params }: WithRouterProps) => {
     });
     const ideaId = idea.data.id;
 
-    clHistory.push({
-      pathname: `/ideas/${idea.data.attributes.slug}`,
-      search: `?new_idea_id=${ideaId}`,
-    });
+    // Check ParticipationMethodConfig for form submission action
+    if (!isNilOrError(phases)) {
+      const phaseParticipationMethod =
+        getLatestRelevantPhase(phases)?.attributes.participation_method;
```

Comment:
Ah that's a good catch... I keep forgetting to test that since it's not working in the BO yet. It should be enough here then to just first check if the URL contains a phase_id (and use that phase if it is), and otherwise we get use the latestRelevantPhase?

Reply:
`latestRelevantPhase` is a strange concept. In the front office the only phase that is relevant is the active phase, because it is not possible to post outside a phase (or continuous project). When I look at the implementation of `latestRelevantPhase`, it is rather complicated. I wonder whether in this file we shouldn't use `getCurrentPhase` instead.